index.html

```diff
@@ -1,12 +1,304 @@
 <!DOCTYPE html>
 <html lang="ru">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
     <title>HTML Academy: Девайс</title>
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
-
+    <!-- Вводная часть сайта -->
+    <header class="page-header">
+      <a class="header-logo" href="https://htmlacademy.ru/">
+        <img
+          class="header-logo-image"
+          src="#"
+          alt="Логотип интернет-магазина 'Device'"
+        />
+      </a>
+      <ul class="user-navigation-list">
+        <li class="user-navigation-item">
+          <input type="text" placeholder="Поиск по сайту" />
+        </li>
+        <li class="user-navigation-item">
+          <a class="navigation-link" href="#">Войти</a>
+        </li>
+        <li class="user-navigation-item">
+          <a class="navigation-link" href="#">Сравнить</a>
+        </li>
+        <li class="user-navigation-item">
+          <a class="navigation-link" href="#">Корзина</a>
+        </li>
+      </ul>
+      <nav class="main-navigation">
+        <ul class="main-navigation-list">
+          <li class="main-navigation-item">
+            <a class="navigation-link" href="catalog.html">Каталог товаров</a>
+          </li>
+          <li class="main-navigation-item">
+            <a class="navigation-link" href="#">Доставка</a>
+          </li>
+          <li class="main-navigation-item">
+            <a class="navigation-link" href="#">Гарантия</a>
+          </li>
+          <li class="main-navigation-item">
+            <a class="navigation-link" href="#">Контакты</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- Основное содержимое страницы -->
+    <main class="page-main">
+      <h1 class="visually-hidden">Интернет магазин "Device"</h1>
+      <!-- Слайдер -->
+      <section class="slider">
+        <h2 class="visually-hidden">Карусель товаров</h2>
+        <img class="slider-image" src="" alt="Пример товара" />
+        <button class="slider-button slider-prev">
+          <span class="visually-hidden">Предыдущее фото</span>
+        </button>
+        <button class="slider-button slider-next">
+          <span class="visually-hidden">Следующее фото</span>
+        </button>
+        <p class="slider-slogan">Порхает как бабочка, жалит как пчела!</p>
+        <p>
+          Этот обычный, на первый взгляд, квадрокоптер оснащен мощным лазером,
+          замаскированным под стандартную камеру.
+        </p>
+        <p><a class="button slider-button" href="#">Подробнее</a></p>
+        <ol class="slider-pagination-list">
```

Comment:
```suggestion
        <ul class="slider-pagination-list">
```
тут можно и лучше обойтись с обычным списком

Reply:
исправила